stdlib/public/core/SetStorage.swift

```diff
@@ -232,6 +233,7 @@ final internal class _SetStorage<Element: Hashable>
 
   @objc
   internal func objectEnumerator() -> _NSEnumerator {
+    _connectOrphanedFoundationSubclassesIfNeeded()
```

Comment:
I feel like these calls in the enumerator callbacks ought to be unnecessary. Shouldn't ObjC methods only be called after you've gone through a bridging function?

Reply:
That's probably true actually, good point